src/sentry/search/eap/span_columns.py

```diff
@@ -621,8 +696,24 @@ def count_processor(count_value: int | None) -> int:
     ),
 }
 
+SPAN_FORMULA_DEFINITIONS = {
+    "http_response_rate": FormulaDefinition(
+        default_search_type="percentage",
+        arguments=[
+            ArgumentDefinition(
+                argument_types={
+                    "string"
+                },  # TODO - this should be an integer, but `resolve_attribute` returns a string
```

Comment:
I think instead of leaving this as a todo we need to add a way to define arguments that aren't just attributes, so that `SearchResolver.resolve_aggregate` knows to cast a user input, which probably would clean up the definition of `http_response_rate` for you too